.github/CODEOWNERS

```
@@ -625,6 +625,7 @@ peps/pep-0743.rst  @vstinner
 peps/pep-0744.rst  @brandtbucher
 peps/pep-0745.rst  @hugovk
 peps/pep-0746.rst  @JelleZijlstra
+peps/pep-0748.rst  @ncoghlan @woodruffw
```

In this case, just the sponsor, because only those with write access can be added (see the red squiggly above).

```suggestion
peps/pep-0748.rst  @ncoghlan
```